### PR TITLE
Created header, CSS and added to general.css

### PR DESCRIPTION
**Pull request by DimitarIM**
Above self-explanatory

### CODE REVIEW

File: css/general.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
```
Comment (Alexander-Lindstroem) [inline css/general.css]: move "margin: 0" to body
padding not required.

Comment (Alexander-Lindstroem) [reply css/general.css]: in body, you can also add min-height: 100%

**Review by Alexander-Lindstroem — CHANGES_REQUESTED** [review]

File: css/general.css (hunk 1 above)
Comment (DimitarIM) [reply css/general.css]: Sure, I've committed the changes.

**Review by Alexander-Lindstroem — APPROVED** [review]
LGTM

**Review by andre84o — APPROVED** [review]
looks good

**Review by andre84o — APPROVED** [review]
LG

**Pull request merged**